.github/workflows/test.yaml

```
@@ -1122,6 +1122,75 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job }} ${{ matrix.platform }}
 
+  driver-adapters:
+    needs: [detect_jobs_to_run]
+    if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)['driver-adapters'] == true }}
+
+    timeout-minutes: 60 # can take longer if platforms are down, so better protect
```

```suggestion
    timeout-minutes: 60 # can take longer if platforms are down, so better protect
```
Note: 60 minutes sounds like a lot to me for these tests 🤔